hapi/hedera-protobuf-java-api/src/main/proto/fees/fee_schedule.proto

```diff
@@ -94,6 +94,11 @@ enum Extra {
     ALLOWANCES=14;
     AIRDROPS=15;
     HOOKS=16;
+    CRYPTO_TRANSFER_TOKEN_FUNGIBLE_COMMON=17;
+    CRYPTO_TRANSFER_TOKEN_NON_FUNGIBLE_UNIQUE=18;
+    CRYPTO_TRANSFER_TOKEN_FUNGIBLE_COMMON_WITH_CUSTOM_FEES=19;
+    CRYPTO_TRANSFER_TOKEN_NON_FUNGIBLE_UNIQUE_WITH_CUSTOM_FEES=20;
+    CRYPTO_TRANSFER_WITH_HOOKS=21;
```

Comment:
This is not one time extra anymore, I have made changes that it is like extra based on number of hook executions